atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -162,7 +166,7 @@ def generate_auto_labels(self, base_labels, df_labels, plugin_labels):
             if lbl not in generated:
                 self.log.warning("requested automatic label %r is not available", lbl)
 
-            elif lbl in plugin_labels:
+            elif lbl in plugin_labels and lbl not in self.auto_always_overwrite:
```

Comment:
Is there any particular reason you can think of not to just *always* overwrite the labels mentioned in `auto_labels`?

Reply:
Given the plugin defaults, always overwriting makes a lot of sense.

Reply:
ok, will always overwrite all auto_labels